tests/bcl-test/Make.frag

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
This file was removed in main (39629ab93b8d723b472a52378615dc5cf03553bd), so why is it added here?

Reply:
Mistake :)